libfwupd/meson.build

```diff
@@ -30,7 +30,7 @@ install_headers([
     'fwupd-plugin.h',
     fwupd_version_h,
   ],
-  subdir: 'fwupd-1/libfwupd',
+  subdir: 'fwupd-2/libfwupd',
```

Comment:
it's a little bit weird to skip `2`, but how would you feel about aligning this with `libfwupd_lt_current`?

Reply:
I think it's a good idea